### PR TITLE
add cugraph-ops

### PR DESCRIPTION
**Pull request by AyodeAwe**
PR adds cugraph-ops to the API section of https://docs.rapids.ai/api.

### CODE REVIEW

File: _data/docs.yml [hunk 1]
```diff
@@ -46,6 +46,16 @@ apis:
       legacy: 1
       stable: 1
       nightly: 1
+  cugraph-ops:
+    name: cuGraph-Ops
+    path: cugraph-ops
+    desc: 'cuGraph-Ops aims to be a low-level, framework agnostic library providing commonly used computational operators for GNNs and other graph operations.'
+    ghlink: https://github.com/rapidsai/cugraph-ops
```
Comment (stadlmax) [inline _data/docs.yml]: cugraph-ops is closed source, hence I am not sure what to do about the link to GitHub. We could either link to cugraph or nothing at all.

File: _data/docs.yml [hunk 2]
```diff
@@ -123,6 +123,16 @@ apis:
...
       stable: 1
       nightly: 1
+  pylibcugraphops:
+    name: cuGraph-Ops
+    path: pylibcugraphops
+    desc: 'cuGraph-Ops aims to be a low-level, framework agnostic library providing commonly used computational operators for GNNs and other graph operations.'
+    ghlink: https://github.com/rapidsai/cugraph-ops
+    versions:
+      # enable or disable links; 0 = disabled, 1 = enabled
+      legacy: 1
```
Comment (ajschmidt8) [inline _data/docs.yml:133]:
```suggestion
      legacy: 0
```

`cugraph-ops` only recently started publishing documentation, so they don't have legacy (`23.02`) docs

File: _data/docs.yml [hunk 3]
```diff
@@ -183,6 +193,16 @@ libs:
...
       stable: 1
       nightly: 1
+  libcugraphops:
+    name: libcugraphops
+    path: libcugraphops
+    desc: 'cuGraph-Ops aims to be a low-level, framework agnostic library providing commonly used computational operators for GNNs and other graph operations.'
+    ghlink: https://github.com/rapidsai/cugraph-ops
+    versions:
+      # enable or disable links; 0 = disabled, 1 = enabled
+      legacy: 1
```
Comment (ajschmidt8) [inline _data/docs.yml:203]:
```suggestion
      legacy: 0
```

`cugraph-ops` only recently started publishing documentation, so they don't have legacy (`23.02`) docs

File: _data/docs.yml [hunk 4]
```diff
@@ -123,6 +123,16 @@ apis:
       legacy: 1
       stable: 1
       nightly: 1
+  pylibcugraphops:
+    name: cuGraph-Ops
+    path: pylibcugraphops
+    desc: 'cuGraph-Ops aims to be a low-level, framework agnostic library providing commonly used computational operators for GNNs and other graph operations.'
+    ghlink: https://github.com/rapidsai/cugraph-ops
```
Comment (ajschmidt8) [inline _data/docs.yml:130]:
```suggestion
```

`cugraph-ops` is a private repository, so we shouldn't post links to it publicly

**Comment by ajschmidt8** [comment]
After chatting with @BradReesWork, he said he is going to eventually work on incorporating `cugraph-ops` docs in the `cugraph` documentation.

So I will close this PR to avoid adding additional pages that would soon be removed.

**Pull request closed without merging**